modules/api/docs/LorisRESTAPI_v0.0.4-dev.md

```diff
@@ -480,7 +481,10 @@ Where the instruments array represents the instruments that were administered fo
 candidate at that visit. InstrumentNames are the short names and the forms for them
 SHOULD all be retrievable through the `project` portion of the API.
 
-PUT / PATCH / POST are not currently supported for candidate instruments.
+POST accepts data of the same format. Any instruments in the Instrument key not currently in the visit test battery will be added to
```

Comment:
Why ask for the Meta object? Just the instruments would be enough no?
```
{
  "Instruments": [...]
}
```

Reply:
The Meta object is generally required in all our endpoints for 1. symmetric 2. as a double check to ensure a bug didn't send it to the wrong endpoint, isn't it?